packages/cozy-client/src/helpers.js

```diff
@@ -1,4 +1,5 @@
 import { Association } from './associations'
+import { URL } from 'url'
```

Comment:
This dep doesn't seem to be in package.json. Is it normal?

Reply:
No it's not. Actually this is needed when running the code in Node but I believe it's not when running it in the browser.
But I don't know how to have different dependencies depending on the target :/.

Reply:
Actually tests are fine without it now. I don't know why it was an issue before.